jobqueue/src/main/java/com/birbit/android/jobqueue/config/Configuration.java

```diff
@@ -130,10 +143,15 @@ public int getThreadPriority() {
         return threadPriority;
     }
 
+    @Nullable
+    public ThreadFactory getWorkerFactory() {
```

Comment:
i do like the worker factory name but since this class uses `consumer` to name these, it will be having 2 names for the same thing. Instead, lets just name this ThreadFactory.